File: output/gemma4-e4b/avatar/Java/Python/atcoder_ABC178_A.py
import sys

def solve():
    # Read the single integer input
    try:
        # Reading from stdin is the standard way to replicate Scanner behavior
        x = int(sys.stdin.read().strip())
    except:
        # Handle case where input might be empty
        return

    # Java logic:
    # bool = (x == 1)
    # result = !bool ? 1 : 0
    # This evaluates to 1 if x != 1, and 0 if x == 1.
    result = 1 if x != 1 else 0
    
    print(result)

solve()

File: output/gemma4-e4b/avatar/Python/Java/atcoder_ABC178_A.java
import java.util.Scanner;

public class atcoder_ABC178_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // The Python code reads a single integer input 'a'
        if (scanner.hasNextInt()) {
            int a = scanner.nextInt();
            
            // Print 1 if a == 0, else 0
            System.out.println(a == 0 ? 1 : 0);
        }
        
        scanner.close();
    }
}